driver/InputFlags.cpp

```diff
@@ -125,7 +129,16 @@ void InputFlags::Parse(int argc, char* argv[])
 
             char short_name = FindShortName(long_name);
 
-            MapInputs[short_name].value = args[i + 1];
+            if ( MapInputs[short_name].is_case_insensitive == true )
```

Comment:
The same should be implemented for "Short Name Input", see line 146 and below.

I recommend factoring out this code into a function or lambda and calling it from here and from the "Short Name Input" branch.

Reply:
@atamazov @DrizztDoUrden -  worked on review comments. thanks.